src/main/java/com/bandwidth/iris/sdk/IrisClientException.java

```diff
@@ -1,42 +0,0 @@
-package com.bandwidth.iris.sdk;
-
-/**
- * Created by sbarstow on 10/10/14.
- */
-public class IrisClientException extends Exception {
-    private String errorCode;
-    private String description;
-
-    public IrisClientException() {
-    }
```

Comment:
An empty IrisClientException is probably of not much use. Consider removing this default constructor. Some more details should always be passed in. Either the "cause" or the error code and description